### PR TITLE
Fix for #19590 - Remove innerHTML method from sample visual

### PR DESCRIPTION
**Pull request by vtkalek**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 This page contains information about changes to the PowerBI Visual Tools (pbiviz).
 
+## pbiviz v1.7.2
```
Comment (AviSander) [inline Changelog.md]: are you bumping the version?

Comment (vtkalek) [inline Changelog.md]: Yes, we agreed to change versions each time we change something

Comment (AviSander) [inline Changelog.md]: so where it the version change? :-)

Comment (vtkalek) [inline Changelog.md]: Yes, i missed it :). Now changed to 1.7.2

**Pull request closed without merging**